hummingbot/strategy/avellaneda_market_making/start.py

```diff
@@ -48,34 +61,33 @@ def start(self):
         self.market_trading_pair_tuples = [MarketTradingPairTuple(*maker_data)]
 
         strategy_logging_options = AvellanedaMarketMakingStrategy.OPTION_LOG_ALL
-        risk_factor = c_map.get("risk_factor").value
-        order_amount_shape_factor = c_map.get("order_amount_shape_factor").value
-
-        execution_timeframe = c_map.get("execution_timeframe").value
-
-        start_time = c_map.get("start_time").value
-        end_time = c_map.get("end_time").value
+        risk_factor = c_map.risk_factor
+        order_amount_shape_factor = c_map.order_amount_shape_factor
 
-        if execution_timeframe == "from_date_to_date":
-            start_time = datetime.datetime.fromisoformat(start_time)
-            end_time = datetime.datetime.fromisoformat(end_time)
+        execution_timeframe = c_map.execution_timeframe_mode.Config.title
+        if isinstance(c_map.execution_timeframe_mode, FromDateToDateModel):
+            start_time = c_map.execution_timeframe_mode.start_datetime
+            end_time = c_map.execution_timeframe_mode.end_datetime
             execution_state = RunInTimeConditionalExecutionState(start_timestamp=start_time, end_timestamp=end_time)
-        if execution_timeframe == "daily_between_times":
-            start_time = datetime.datetime.strptime(start_time, '%H:%M:%S').time()
-            end_time = datetime.datetime.strptime(end_time, '%H:%M:%S').time()
+        elif isinstance(c_map.execution_timeframe_mode, DailyBetweenTimesModel):
+            start_time = c_map.execution_timeframe_mode.start_time
+            end_time = c_map.execution_timeframe_mode.end_time
```

Comment:
Would it be possible to make the make the methods to return the times be called the same way? If we make them polymorphic there will be no need to check for the type of `c_map.execution_timeframe_mode`

Reply:
Addressed